src/FileMigrationHelper.php

```diff
@@ -102,6 +102,17 @@ protected function migrateFile($base, File $file, $legacyFilename)
             return false;
         }
 
+        // If this file has the wrong class for it's extension, update the classname
+        $destinationClass = $file->get_class_for_file_extension($extension);
```

Comment:
I would have move this block of code to a private function just for readability.

Reply:
I'm not sure there's a elegant way of unit testing this. And it's affecting a part of code that's probably not going to be used very much. So not having unit tests for this is probably kosher.

Reply:
Moved and yea, I don't think a unit test is necessary.